buildpacks/nodejs-yarn/src/snapshots/errors___yarn_cli_layer_download_error.snap

```diff
@@ -2,7 +2,8 @@
 source: buildpacks/nodejs-yarn/src/errors.rs
 ---
 - Debug Info:
-  - HTTP error while downloading file: http: invalid format
+  - Request to https://test/error failed
+    Error: error sending request for url (https://test/error)
```

Comment:
Do we have any control over the middleware error formatting? I like that the URL is wrapped in character's but it's a slightly different convention from our other locations (backticks). Not a deal breaker, but if it isn't too time-consuming it would look more consistent

Reply:
No control over the error formatting, no. You'll get whatever the library provides for messages which is why this is in the debug info section.